### PR TITLE
Added border color for areas

### PR DESCRIPTION
**Pull request by rokuz**

### CODE REVIEW

**Comment by therearesomewhocallmetim** [comment]
LGTM

**Comment by Komzpa** [comment]
guys, border color for areas is called 'casing' in MapCSS and is already there :)

**Comment by rokuz** [comment]
Thank you. Do you mean that I have to rename 'border-color' to 'casing'? No problem. But what do you mean by "already there"? No line in libkomwm.py writes in _area.border_ before.

**Comment by Komzpa** [comment]
There's a common spec for all this - http://wiki.openstreetmap.org/wiki/MapCSS/0.2 - have a look there each time you'd like to reinvent something :)

By MapCSS design, what you call 'border' here is in fact the linear geometry of feature.

In mapsme context, in previous incarnations it was encoded separately from polygon, so the 'casing-width' and casing-color were just automatically promoted to render the border - the linear geometry. There wasn't area.border around, because it was supported via LineRuleProto().

If your refactoring led to area.border being used, just fill it from casing-* fields as in https://github.com/rokuz/kothic/blame/6c1f59ca6e188cb05e5177447e7ef747941eed0c/src/libkomwm.py#L332 (note the commented out `or st.get('fill-color'))` suggesting this branch was used in polygon rendering too :)

**Comment by rokuz** [comment]
To know whether i reinvent something i have to know whether it has been invented in mapsme context :) Anyway the link may be useful, appreciate it.

**Comment by rokuz** [comment]
@Komzpa How do you feel about this implementation?

**Comment by Komzpa** [comment]
Now it's compatible with other implementations, thanks :)

Just in case, you can check mapcss styles in JOSM or galileo's mobile https://getyourmap.com/our_technology/#mapcss or any other compatible engine.

**Comment by rokuz** [comment]
@Zverik take a look and merge pls

File: src/libkomwm.py [hunk 1]
```diff
@@ -345,6 +345,10 @@ def addPattern(dashes):
                             dr_line.join = dr_linejoins.get(st.get('casing-linejoin', 'round'), ROUNDJOIN)
                             dr_element.lines.extend([dr_line])
 
+                        if ('fill-color' in st) and (float(st.get('fill-opacity', 1)) > 0):
+                            dr_element.area.border.color = mwm_encode_color(colors, st, "casing")
+                            dr_element.area.border.width = st.get('casing-width')
```
Comment (Zverik) [inline src/libkomwm.py]: Я выше и ниже вижу, что ширина умножается на `WIDTH_SCALE`. Можешь и тут?

**Review by Zverik — CHANGES_REQUESTED** [review]

File: src/libkomwm.py (hunk 1 above)
Comment (rokuz) [reply src/libkomwm.py]: Могу, если надо :) В рендеринге мы все равно не учитываем заданную ширину :)

Comment (rokuz) [reply src/libkomwm.py]: @Zverik done PTAL

**Pull request merged**